lib/immutable-struct.rb

```diff
@@ -81,8 +81,25 @@ def self.new(*attributes,&block)
         end
       end
 
+      @@aliases = {}
+
+      def self.alias_attribute(aliaz, attr, &block)
+        @@aliases.store(aliaz, [attr, block])
```

Comment:
This is probably "primitive obsession", but I had a hard time convincing myself that we really needed a custom type for this data... A likely causing factor of primitive obsession 😅 💦